bin/pycbc_optimize_snr

```diff
@@ -0,0 +1,290 @@
+
+netsnr = 0
+for idx, ifo in enumerate(ifos):
+    coinc_results['foreground/'+ifo+'/mass1'] = mass1
+    coinc_results['foreground/'+ifo+'/mass2'] = mass2
+    coinc_results['foreground/'+ifo+'/spin1z'] = spin1z
+    coinc_results['foreground/'+ifo+'/spin2z'] = spin2z
+    coinc_results['foreground/'+ifo+'/f_lower'] = flow
+    coinc_results['foreground/'+ifo+'/window'] = 0.1
+    coinc_results['foreground/'+ifo+'/sample_rate'] = int(sample_rate)
+    coinc_results['foreground/'+ifo+'/template_id'] = 0
+    coinc_results['foreground/'+ifo+'/template_duration'] = \
+        fp['template_duration'][()]
+    followup_data[ifo] = {}
+    fseries = interpolate(data[ifo].psd, 0.25)
+    followup_data[ifo]['psd'] = fseries
+    followup_data[ifo]['snr_series'] = snr_series_dict[ifo]
+
+    # Find loudest SNR within coincidence window of the largest peak
+    snr_peak = abs(loudest_snrs[ifo])
+    # And check light travel time
+    lttbd = Detector(ifo).light_travel_time_to_detector(Detector(loudest_ifo))
+    # Add small buffer
+    lttbd += 0.005
+
+    time_window = [loudest_snr_time_allifos-lttbd,
+                   loudest_snr_time_allifos+lttbd]
+
+    snr_slice = snr_series_dict[ifo].time_slice(time_window[0], time_window[1])
+    max_snr_idx = numpy.argmax(abs(snr_slice))
+    loudest_snr_time = snr_slice.sample_times[max_snr_idx]
+    loudest_snr = snr_slice[max_snr_idx]
+
+    coinc_results['foreground/'+ifo+'/end_time'] = loudest_snr_time_allifos
+    coinc_results['foreground/'+ifo+'/end_time'] = loudest_snr_time
+    coinc_results['foreground/'+ifo+'/snr_series'] = snr_series_dict[ifo]
+    coinc_results['foreground/'+ifo+'/psd_series'] = data[ifo].psd
+    coinc_results['foreground/'+ifo+'/delta_f'] = delta_f
+    coinc_results['foreground/'+ifo+'/event_id'] = \
+        'sngl_inspiral:event_id:'+str(idx)
+    coinc_results['foreground/'+ifo+'/snr'] = abs(loudest_snr)
+    netsnr += abs(loudest_snr)**2
+    coinc_results['foreground/'+ifo+'/sigmasq'] \
+        = snr_series_dict['sigmasq_' + ifo]
+    coinc_results['foreground/'+ifo+'/coa_phase'] \
+        = numpy.angle(loudest_snr)
+    coinc_results['foreground/'+ifo+'/chisq'] = 1.
+    coinc_results['foreground/'+ifo+'/chisq_dof'] = 1
+coinc_results['foreground/stat'] = numpy.sqrt(netsnr)
+coinc_results['foreground/ifar'] = fp['ifar'][()]
+
+channel_names = {}
+for ifo in fp['channel_names'].keys():
+    channel_names[ifo] = fp['channel_names'][ifo][()]
+
+kwargs = {'psds': {ifo: followup_data[ifo]['psd'] for ifo in ifos},
+          'low_frequency_cutoff': flow,
+          'followup_data': followup_data,
+          'channel_names': channel_names}
+comments = ['Automatic PyCBC followup to find largest SNR parameters']
+
+doc = live.SingleCoincForGraceDB(coinc_ifos, coinc_results,
+                                 upload_snr_series=True,
+                                 followup_ifos=fup_ifos, **kwargs)
+gracedb_server = 'https://gracedb-playground.ligo.org/api/'
```

Comment:
@spxiwh did you intend this functionality to actually upload to GraceDB playground, or was this just test code?

Reply:
I think even if this is intended, I recommend having an actual command line option for this, with a default to gracedb-playground (see the `pycbc_live` args).

Reply:
Yes, definitely, this should be a command line. In production this should upload to the production server.